tracer/src/Datadog.Trace/DatabaseMonitoring/DatabaseMonitoringPropagator.cs

```diff
@@ -17,8 +17,10 @@ internal static class DatabaseMonitoringPropagator
         private const string SqlCommentVersion = "ddpv";
         private const string SqlCommentEnv = "dde";
```

Comment:
e.g,
```suggestion
        private const string SqlCommentEnv = "dde";
        internal const string DbmsPrefix = "/*{SqlCommentSpanService}='";
```

and below becomes:

```csharp
propagatorSringBuilder.Append(DbmsPrefix).Append(Uri.EscapeDataString(context.ServiceNameInternal)).Append('\'');
```

As a small nit, you _shouldn't_ use interpolated strings `$"{something}"` with `StringBuilder`. Using interpolated strings creates extra garbage (which using `StringBuilder` is meant to prevent 🙂 

So instead of this:

```csharp
propagatorSringBuilder.Append($",{SqlCommentEnv}='{Uri.EscapeDataString(envTag)}'");
```

do this:

```csharp
propagatorSringBuilder.Append(',').Append(SqlCommentEnv).Append("='").Append(Uri.EscapeDataString(envTag)).Append('\'');
```

Note that this also uses the "native" format for variables, instead of converting them to strings (`char` in this case, but applies to `int`s etc as well)